coreos-base/hard-host-depends/hard-host-depends-0.0.1.ebuild

```diff
@@ -163,6 +163,11 @@ RDEPEND="${RDEPEND}
 	virtual/cdrtools
 	"
 
+# Host dependencies for pre-populating rkt container images
+RDEPEND="${RDEPEND}
+	app-emulation/rkt
```

Comment:
Right, not as it is now. It was still convenient to test the wrappers within the SDK, however that doesn't justify the hard host dependency. Should I remove it?